server/WebAPI/Services/NewsService.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using WebAPI.Repositories;
+using WebAPI.DTO;
+using Entities.Models;
+using AutoMapper;
+using WebAPI.Validation;
+using FluentValidation;
+
+namespace WebAPI.Services
+{
+    public class NewsService
+    {
+        private readonly NewsRepository _newsRepository;
+        private readonly IMapper _mapper;
+
+        public NewsService(NewsRepository _newsRepository, IMapper _mapper)
+        {
+            this._newsRepository = _newsRepository;
+            this._mapper = _mapper;
+        }
+        public IEnumerable<NewsDTO> GetAllNews()
+        {
+            var news = _newsRepository.GetAllNews();
+            var validator = new ValidateNews();
+            foreach (var newsElement in news)
+            {
+                try 
+                {
+                    validator.ValidateAndThrow(newsElement);
+                }
+                catch(ValidationException ex)
+                {
+                    Console.WriteLine(ex.Message);
+                    news = news.Where(el => el != newsElement);
+                }
+            }
```

Comment:
you don't need validate manually db entities it makes EntityFramework, also validators are for put and add database queries, you can't get invalid data from database

when i spoke about validation in get request from client i mean situation when client requests some data with some parameter and that parameter could be invalid, such as unexciting news id when client wants certain news item

Reply:
deleted useless validation